evap/static/scss/components/_grid.scss

```diff
@@ -1,5 +1,5 @@
 .container {
-    padding-top: 80px; // for navbar (and breadcrumbs)
+    padding-top: 15px; // to have some space between content and navbar/breadcrumbs
     flex: 1 0 auto;
 }
```

Comment:
Cool that `sticky` is working out of the box :)
- I’d replace `padding-top` with `.pt-3` directly on the `.container` (slightly different in height, but I guess you aren’t attached to the exact amount of `15px`)
- According to my firefox, `flex` has no effect on this element as it is not flexible, so the wohle stylesheet rule can be removed (unless I have overseen a different usage of `.container`, but my search only deteced one in `base.html`)